wordpress-mobile/WordPress-iOS
# Update WordPress app icons

Fixes #15854. This PR updates the app icons to the new blue color.

<img width="106" alt="Screenshot 2021-04-02 at 10 47 05" src="https://user-images.githubusercontent.com/4780/113404870-c55a1e00-93a0-11eb-84a6-6a1ee05b7137.png">

@mattmiklic I replaced all the existing icons, but I noticed there were a couple we didn't have already – 20pt @1x and @2x:

<img width="195" alt="Screenshot 2021-04-02 at 10 32 00" src="https://user-images.githubusercontent.com/4780/113405057-179b3f00-93a1-11eb-96ef-989d8177d95c.png">
<img width="205" alt="Screenshot 2021-04-02 at 10 31 55" src="https://user-images.githubusercontent.com/4780/113405061-19650280-93a1-11eb-8a0a-36b3eeb354d0.png">

They weren't there already with our previous icon set, but I guess it might be good to add them for completion? I wasn't sure if I could use the 40pt@1x icon in place of 20pt@2x.

**To test**

- Check the AppImages.xcasset asset catalog and ensure that all of the AppIcon and AppIcon-Internal icons are the new blue
- Build and run on iPhone, iPad, and 12.9" iPad and check that the app icon appears correctly. You can also check in system Settings or by doing a spotlight search that those icons are correct too.

## Regression Notes

1. Potential unintended areas of impact

None, just updating image assets.

PR submission checklist:
